### DISCUSSION TITLE
US Series Battery Support

### DISCUSSION THREAD

**Discussion by michaeljaddy** [mletenay/home-assistant-pylontech#5] [Ideas]
Would it be possible for this integration to be modified to support US series (2000/3000/5000) batteries?

I can get the stream server to connect to the batteries and the integration tries to connect to the stream server however it throws an error (assuming the commands on the high voltage batteries are different from those on low voltage)

**Comment by guybw** [comment]
Good evening,
I started looking at this today with @michaeljaddy

I think the problem is that some entries in info are not showing up on the US series devices.
![image](https://github.com/mletenay/home-assistant-pylontech/assets/64648444/0ed249d0-e945-4b69-9351-44ce1a4b2384)


I started to comment out some of the links in class InfoCommand: that we don't have but I ended up not even logging any more.

Could you give us any pointers on how to go with this at all?
Thanks for the great work on this!
